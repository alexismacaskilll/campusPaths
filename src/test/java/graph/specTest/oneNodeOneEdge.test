# Tests that a graph can have a self-edge, a node who has itself as one of its children

# Create a graph
CreateGraph graph1

# Add a node to the graph
AddNode graph1 n1

# Add a self edge to the graph
AddEdge graph1 n1 n1 label

# Print the outgoing edges from n1
ListChildren graph1 n1