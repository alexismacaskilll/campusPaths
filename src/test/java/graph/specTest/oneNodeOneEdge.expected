# Tests that a graph can have a self-edge, a node who has itself as one of its children

# Create a graph
created graph graph1

# Add a node to the graph
added node n1 to graph1

# Add a self edge to the graph
added edge label from n1 to n1 in graph1

# Print the outgoing edges from n1
the children of n1 in graph1 are: n1(label)